website/src/components/SearchPage/fields/DateRangeField.spec.tsx

```diff
@@ -130,12 +129,8 @@ describe('DateRangeField', () => {
         );
     });
 
-    it('updates query params if user types in new dates', async () => {
```

Comment:
A test case that checks clearing is great, but the original one seems more important to me. Would it be possible to check user typing as well?